### ISSUE TITLE
Collapsed group after map loads

### ISSUE THREAD

**Issue by jaitor1** [3liz/lizmap-web-client#192]
Hi,

I have several groups in one particular map and I would like all of them to load collapsed. Is there way to do so? 

If not, it would be a nice enhancement...( a new "Collapsed" checkbox in lizmap plugin just for groups)

**Comment by rldhont** [comment]
To do so, before we enhance web client and plugin, you can use a JavaScript in the media folder, like the file shown in fb0ba36 , see doc https://github.com/3liz/lizmap-documentation/blob/master/source/publish/advanced_lizmap_config.rst#adding-your-own-javascript

**Comment by rmarzocchi84** [comment]
Have a look also to this issue https://github.com/3liz/lizmap-web-client/issues/245. 

```
lizMap.events.on({
uicreated: function(e) {
$('#group-groupname td a.expander').click();
}
});
```

I think you can close the issue

**Comment by Gustry** [comment]
IMHO, let's keep this ticket open. This should be included in LWC core, not having to use javascript.

**Comment by rmarzocchi84** [comment]
You are right!

**Comment by Gustry** [comment]
If I'm correct, this already done using a QGIS theme, and/or there is some progress in LWC 3.7

CC @nboisteault ?

**Comment by nboisteault** [comment]
This behavior will be available in LWC 3.7 by defining a theme with all groups collapsed and checking `Activate first map theme on startup`.
